### ISSUE TITLE
Very high -px for top on textarea causes repaint glitch in Chrome

### ISSUE THREAD

**Issue by matthew-dean** [ajaxorg/ace#2362]
Here's my scenario:

I have a custom theme for an Ace editor instance. I noticed right away that there was a "glitch", a quick 1-pixel shift up and down (within 100 ms) every time I selected a line.

When I tried the default theme: no glitch.
When I changed the line-height of my custom theme to match the default theme: no glitch.

After surveying everything that changed when selected, I narrowed the glitch trigger to the textarea element. I intuited that probably the large value (-10000000px) that the textarea was set to was causing a repaint glitch that only happened in conjuction with a line-height that didn't match pixel height, or some weird combo like that.

Sure enough, I went into Ace and saw that old IE had a different top setting:

```
if (useragent.isOldIE) text.style.top = "-100px";
```
So, I changed the non-IE value also to -100px:
```
this.focus = function() {
        text.style.position = "fixed";
        text.style.top = "-100px";
        text.focus();
        setTimeout(function() {
            text.style.position = "";
        }, 0);
    };
```
It worked. No glitch. 

I did some testing, and even a value of negative 1 million instead of 10 million works, which PROBABLY means (I'm guessing) that when an integer value is being exceeded, it's causing a repaint. It's possibly always causing a repaint, but that repaint may only be visible in some cases.

Long and short, my recommendation for positioning things off-screen is to be conservative with negative values, so that you don't exceed internal variable calculations. In this case, anything from -100px to -1000000px didn't have the issue, but I would use the smallest number required to have expected functionality.

**Comment by matthew-dean** [comment]
By the way, this is in Chrome 38.0.2125.104.

**Comment by nightwing** [comment]
Thank you for debugging this! I've added the fix you found in 16b542a .

**Comment by matthew-dean** [comment]
Hey, thank you for dropping it in! It's probably Chrome that's at fault, but it's nice to have a workaround.
:-)
